code/src/components/Checkbox.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import "./form.css";
+import "./checkbox.css";
+
+const Checkbox = ({ isBoxChecked, onCheckBoxChange }) => {
+
+  return (
+    <section className="section-container">
+      <h2 tabIndex="0">I would like a price estimation for:</h2>
+      <label htmlFor="moving">
+        <input
+          tabIndex="0"
+          type="checkbox"
+          id="moving"
+          onChange={() => onCheckBoxChange("moving")}
+          checked={isBoxChecked.includes("moving")}
+        >
+        </input>
```

Comment:
`<input />` is self closing tag, no need to have an opening and closing tag separate